app/views/late_reservation_mailer/delinquent_reservation.html.slim

```diff
@@ -0,0 +1,7 @@
+h2 Your gifts were due today (#{Date.today.strftime("%B %d, %Y")}).
+p
+  | At least one of the gifts you reserved has not yet been received by Wediko.
+    If you shipped them recently, there's no reason to worry, but if you have yet to ship your gifts, please log in below to mark them as shipped.
```

Comment:
Should this also include a note about unreserving them? That way Wediko would have a bit more time to step in and order gifts the abandoned reservations